packages/panna-sdk/src/core/onramp/onramp.ts

```diff
@@ -189,3 +191,67 @@ export function getOnrampProviders(countryCode: string): ProviderInfo[] {
     COUNTRY_PROVIDER_MAP[normalizedCountryCode] || [];
   return providers.map((provider) => PROVIDERS[provider]);
 }
+
+/**
+ * Fetches fiat prices for a specific token or all tokens on a given chain
+ *
+ * @param params - Parameters for fetching token fiat prices
+ * @param params.chainId - The chain ID of the token
+ * @param params.tokenAddress - The address of the token (optional) if not provided, fetches prices for all tokens on the chain
```

Comment:
For consistent typedoc styling.
```suggestion
 * @param params.tokenAddress - (Optional) The address of the token to fetch the prices for. (Default: Fetches prices for all the tokens on the chain)
```